.gitignore

```diff
@@ -48,3 +48,4 @@ OpenCL-Headers/
 target/
 .venv/
 docs/build/
+publish.sh
```

Comment:
is this by accident?

Reply:
No, this might be tight to my configuration but this is the local script I use to sync all repos. Shall we take it out?

Reply:
no problem from my side.